realsense2_camera/src/realsense_node.cpp

```diff
@@ -1091,7 +1097,12 @@ void RealSenseNode::setupStreams()
             _enable[INFRA2])
         {
             static const char* frame_id = "depth_to_infra2_extrinsics";
-            auto ex = getRsExtrinsics(DEPTH, INFRA2);
+            // DEPTH -> INFRA2 query seems broken with librealsense ver >= 2.36.0
+            // This is a quick fix for now until Intel provides an offical fix
+            auto ex = getRsExtrinsics(INFRA2, DEPTH);
+            ex.translation[0] *= -1;
+            ex.translation[1] *= -1;
+            ex.translation[2] *= -1;
```

Comment:
I'm not sure what the data type here is, just checking in that there isn't a rotation that also needs reversing here?

Reply:
For rotation the output was identity matrix so it should be ok.